src/voice-pe/using-the-device/new-ha-instance

```diff
@@ -0,0 +1,45 @@
+---
+zendesk:
+  article_id: 30670495779997
+  name: Connecting your Home Assistant Voice PE to a new Home Assistant Instance
+  position: 10
+  labels: voice pe, operation, using the device
+---
+
+When connecting your Home Assistant Voice Preview Edition (VPE) device to a new Home Assistant instance, a factory reset is required. This is because the VPE device stores an encryption key tied to the original Home Assistant instance, even if the device is removed from that setup.
+A factory reset removes this key and allows the device to securely pair with a new instance. This may be necessary if you’ve reinstalled Home Assistant from scratch, are setting up a new environment, or want to use the device with a different system.
+
+## To reset the device
+
+{% steps %}
+{% step "Removing the device from Home Assistant (if accessible)" %}
+{% image "/static/img/voice-pe/voice_delete_integration_instance.png" "Screenshot showing where to remove the integration" %}
+{% stepContent %}
+
+- If you still have access to the Home Assistant instance the device was previously connected to, follow the [removing an integration instance](https://www.home-assistant.io/common-tasks/general/#removing-an-integration-instance).
+
+*This step is optional but recommended before resetting the device.*
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Press the button until it lights up red (22 seconds)" %}
```

Comment:
we already have a reset procedure. should we link to that, instead of creating another version of it here.
https://support.nabucasa.com/hc/en-us/articles/25800178374557-Resetting-Home-Assistant-Voice-Preview-Edition